azure-pipelines.yml

```diff
@@ -86,7 +86,7 @@ stages:
       displayName: 'Install Node.js 10.16.3'
     - script: yarn install
       displayName: 'Run yarn install'
-    - script: yarn gulp release --android
+    - script: yarn gulp debug-release --android
```

Comment:
Now we need to have both here, debug and non debug build like in Windows:
```
    - script: yarn gulp release --android
      displayName: 'Run yarn release for android'
      condition: and(succeeded(), eq('${{ parameters.releaseBuild }}', true))
    - script: yarn gulp debug-release --android
      displayName: 'Run yarn debug release for android'
      condition: and(succeeded(), eq('${{ parameters.releaseBuild }}', false))
```

Reply:
The problem is that part of the build of the android application bundle for the Google Play store is to sign the bundle with a private key. This cannot easily moved to azure without compromising the key, so I have opted to be doing this on a physical machine for now.
There are some more constraints with the Google Play store around versioning that make it hard to fully automate builds to be published in this way, so doing this manually and locally is probably best anyway.

Reply:
I don' t understand exactly... I understand the part that we need a private key, but I don't understand why we can't have the two options (release and debug-release) like in windows. I understand that by default it will execute the debug-release anyway.
I suppose that what you say is that the release will not work because it simply does not have the private key so is better to not put the option here?
If that is the reason, then what we do is to use different private keys for the "official" release and the "official" nightly, what does not seem very clean. The user will need to remove one to install the other. There must be a way to use, in a safe way, the same private key for both versions. Another thing is a local build of the developer, in this case of course the private key will be different only for testing.